### PR TITLE
add case for DeathPactException stopping actor

### PR DESCRIPTION
**Pull request by onsails**
See https://github.com/akka/akka/blob/master/akka-actor/src/main/scala/akka/actor/FaultHandling.scala#L155

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

**Review by johanandren — APPROVED** [review]

File: akka-docs/rst/java/fault-tolerance.rst [hunk 1]
```diff
@@ -63,6 +63,7 @@ exceptions are handled by default:
 
 * ``ActorInitializationException`` will stop the failing child actor
 * ``ActorKilledException`` will stop the failing child actor
+* ``DeathPactException`` will stop the failing child actor
```
Comment (patriknw) [inline akka-docs/rst/java/fault-tolerance.rst]: same thing in akka-docs/rst/scala/fault-tolerance.rst

**Review by patriknw — CHANGES_REQUESTED** [review]

File: akka-docs/rst/java/fault-tolerance.rst (hunk 1 above)
Comment (prettynatty) [inline akka-docs/rst/java/fault-tolerance.rst]: done

**Review by patriknw — APPROVED** [review]
LGTM, thanks

**Pull request merged**